make.ps1

```diff
@@ -113,6 +113,8 @@ elseif ($command -eq "version")
 		{
 			$replacement = (gc $mod) -Replace "Version:.*", ("Version: {0}" -f $version)
 			sc $mod $replacement
+			$replacement = (gc $mod) -Replace "modchooser:.*", ("modchooser: {0}" -f $version)
```

Comment:
You should probably ensure the `\t` exists in both strings.
The replacement above should also do this, but you aren't obligated to change that.